spec/unique_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe Unique do
+  context '.reset!' do
```

Comment:
This needs to be added to `spec/unique/unique_spec.rb` instead of here so CI will run this.  It looks good otherwise. ```https://travis-ci.org/b264/unique/jobs/164806377#L165```